controllers/cartController.js

```diff
@@ -0,0 +1,15 @@
+const cartService = require("../services/cartService");
+const { catchAsync, raiseCustomError } = require("../utils/error");
+
+const getCartByUserId = catchAsync(async (req, res) => {
+  const userId = req.user.id;
+  if (!userId) {
+    raiseCustomError("KEY_ERROR", 400);
+  }
+
+  const cart = await cartService.getCartByUserId(userId);
```

Comment:
```suggestion
  const carts = await cartService.getCartByUserId(userId);
```
userId로 필터링하면 장바구니가 하나가 아닐 수 있습니다. 정확한 네이밍을 하는 습관을 길러주세요.

Reply:
네! 알겠습니다. 네이밍할 때 구체적인 것까지 생각하겠습니다.